docs/plugins.rst

```diff
@@ -167,7 +167,7 @@ definitions in Airflow.
     from airflow.models.baseoperator import BaseOperatorLink
     from airflow.operators.gcs_to_s3 import GoogleCloudStorageToS3Operator
     from airflow.sensors.base_sensor_operator import BaseSensorOperator
-    from airflow.executors.base_executor import BaseExecutor
+    from airflow.executors.base_executors import BaseExecutor
```

Comment:
This counds as a breaking change - someone has probably created a custom Executor out there. Is it possible not not change this module name please?

Reply:
It was somehow a mistake. There is no base_executors package :). Fixed